Realm/Realm/Realm.csproj

```diff
@@ -1,4 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Target Name="ExtractChangelogToPack" BeforeTargets="GenerateNuspec" Condition="'$(CI)' == 'true'">
+    <Message Text="Extracting release note" />
+    <ExtractChangelog FilePath="$(ProjectDir)\..\..\CHANGELOG.md">
+      <Output TaskParameter="ExtractedText" PropertyName="PackageReleaseNotes" />
+    </ExtractChangelog>
+  </Target>
+
+  <Target Name="WriteChangelogToFile" AfterTargets="ExtractChangelogToPack">
+    <WriteLinesToFile File="ExtractedChangelog.md" Lines="$(PackageReleaseNotes)" Overwrite="true" Encoding="Unicode" />
+  </Target>
```

Comment:
Can we move this toward the bottom as well or is it important that they're at the top?